test-worker.sh

```diff
@@ -0,0 +1,8 @@
+# Test working for docker < max & dependency < max
+./venv/bin/python worker/codalabworker/main.py \
+ --server https://worksheets.codalab.org \
```

Comment:
For the sake of travis testing, we should run this worker against a local server, or maybe make this bash script take the server as an argument so we can run this worker against local dev in travis.